src/components/Note/EditableNoteMentions.tsx

```diff
@@ -0,0 +1,187 @@
+import uniqueId from 'lodash.uniqueid';
+import React, { FC, useEffect, useState } from 'react';
+import Tribute, { TributeItem } from 'tributejs';
+import Button from '../Button/Button';
+import ButtonToolbar from '../Button/ButtonToolbar';
+import Card from '../Card/Card';
+import CardBody from '../Card/CardBody';
+import FormLabelGroup from '../Form/FormLabelGroup';
+import Input from '../Input/Input';
+import { Note } from './Note.types';
+import NoteHeader from './NoteHeader';
+
+type MentionableUser = {
+  key: string;
+  value: string;
+  email: string;
+};
+
+function injectTribute(mentionableUsers: MentionableUser[], noteBodyId: string) {
+  if (mentionableUsers) {
+    const tributeStyles = `
+      .tribute-container {
+        position: absolute;
+        top: 0;
+        left: 0;
+        height: auto;
+        display: block;
+        max-width: none !important;
+        box-shadow: 0 0 1px #a0a0a0, 0 0 1.5em rgb(0 0 0 / 20%), inset 0 1px 0 #fff;
+        border-radius: 4px;
+        z-index: 999999;
+        overflow: auto;
+        max-height: 300px;
+      }
+
+      .tribute-container ul {
+        margin: 0;
+        padding: 0;
+        list-style: none;
+        background: #efefef;
+      } 
+
+      .tribute-container li {
+        padding: 8px;
+        cursor: pointer;
+      }
+
+      .tribute-container span {
+        font-weight: bold;
+      }
+
+      .tribute-container .menu-highlighted {
+        font-weight: bold;
+      }
+
+      .note__mention-highlight {
+        background: #5995db !important;
+        color: #e3e3e3 !important;
+      }
+      
+      .note__mention-email {
+        font-weight: lighter !important;
+        padding-left: 20px;
+        padding-right: 3px;
+        float: right;
+        color: inherit;
+      }
+    `;
+    const styles = document.createElement('style');
```

Comment:
Instead of appending styles like this, we already make use of `styled-jsx` elsewhere in this library. Perhaps we could do that here to keep things consistent.